lib/Common/Memory/arm64/arm64_SAVE_REGISTERS.asm

```diff
@@ -29,23 +27,22 @@
 
     ; Can't use sp with stp so mov to a volatile register
     ; and then store onto passed in array
-    mov   x16, sp
```

Comment:
>x16 [](start = 10, length = 3)

@agarwal.sandeep@microsoft.com this is another place where x16 was used as a temp register (as is allowed by spec). In my opinion, with respect to #4276, x16 should be marked as RA_DONTALLOCATE to avoid problems in the future.

Reply:
#4279 changed it to r17. Will see if we need to change it.